File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-apply-styles-to-ranges-in-workbooks_2.cs
            Microsoft.Office.Tools.Excel.NamedRange rangeStyles =
                this.Controls.AddNamedRange(this.Range["A1"], "rangeStyles");

            rangeStyles.Value2 = "'Style Test";
            rangeStyles.Style = "NewStyle";
            rangeStyles.Columns.AutoFit();

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-apply-styles-to-ranges-in-workbooks_2.vb
        Dim rangeStyles As Microsoft.Office.Tools.Excel.NamedRange = _
            Me.Controls.AddNamedRange(Me.Range("A1"), "rangeStyles")

        rangeStyles.Value2 = "'Style Test"
        rangeStyles.Style = "NewStyle"
        rangeStyles.Columns.AutoFit()